.gitignore

```diff
@@ -1,12 +1,10 @@
 .DS_Store
 
-!dist/
-dist/*
-!dist/minecraftBedrock
-dist/minecraftBedrock/*
-!dist/minecraftBedrock/fileDefinitions.json
+dist
 
 .env
 packages.zip
 dataPackage.ts
-scripts/vanillaData/raw
\ No newline at end of file
+scripts/vanillaData/raw
+
+deno.lock
```

Comment:
Lock files are important in open-source projects to ensure consistent versioning between users.